### PR TITLE
Add failing test for #105

### PR DESCRIPTION
**Pull request by chrisblossom**
See https://github.com/sindresorhus/globby/issues/105

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -214,3 +214,29 @@ test('respects gitignore option false - sync', t => {
...
+test.failing('throws ENOTDIR when specifying a file as cwd - async', async t => {
+	t.plan(1);
+
+	const isFile = path.resolve('fixtures/gitignore/bar.js');
+
+	try {
+		await m('.', {cwd: isFile});
+	} catch (error) {
+		t.is('ENOTDIR', error.code);
+	}
```
Comment (sindresorhus) [inline test.js]: Use `t.throwsAsync` instead of try/catch. Then you can drop the `t.plan()`.

File: test.js [hunk 2]
```diff
@@ -214,3 +214,29 @@ test('respects gitignore option false - sync', t => {
...
+test.failing('throws ENOTDIR when specifying a file as cwd - sync', t => {
+	t.plan(1);
+
+	const isFile = path.resolve('fixtures/gitignore/bar.js');
+
+	try {
+		m.sync('.', {cwd: isFile});
+	} catch (error) {
+		t.is('ENOTDIR', error.code);
+	}
```
Comment (sindresorhus) [inline test.js]: Use `t.throws` instead of try/catch. Then you can drop the `t.plan()`.

File: test.js (hunk 1 above)
Comment (chrisblossom) [reply test.js]: Done.

File: test.js (hunk 2 above)
Comment (chrisblossom) [reply test.js]: Done.

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**